### PR TITLE
Updated the styles of the container commands

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

This PR uses comment() which hasn't been merged yet. WIP PR at #15964

This command defines a lot of options, so I don't know if I've updated all the possible outputs:

container_params

container_tags

container_one_service

container_select_service

container_one_parameter

container_tagged_services

### CODE REVIEW

**Comment by fabpot** [comment]
What finishing #15964? That would allow me to merge some of the PRs.

**Comment by fabpot** [comment]
fabbot has some good suggestions

**Comment by javiereguiluz** [comment]
Fixed fabbot issues.

**Comment by fabpot** [comment]
Can you rebase as tests were badly broken where run? Thanks.

**Comment by javiereguiluz** [comment]
Rebased.

**Comment by fabpot** [comment]
There are some failing tests.

**Comment by javiereguiluz** [comment]
I've fixed the tests (at least locally).

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php [hunk 1]
```diff
@@ -245,71 +246,79 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
...
+                    }
                 }
             }
         } else {
-            $description[] = '<comment>Tags</comment>             -';
+            $tagInformation = '-';
         }
+        $tableRows[] = array('Tags', $tagInformation);
+
+        $tableRows[] = array('Scope', $definition->getScope());
```
Comment (fabpot) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php]: You should pass `false` to `getScope` to avoid the deprecation notice.

Comment (javiereguiluz) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php]: Done. Thanks.

**Comment by fabpot** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**